source/spawn/spawn-channels.adb

```diff
@@ -710,19 +710,29 @@ package body Spawn.Channels is
            (Self.Stdin_Child,
             Self.Stdin_Parent,
             Success);
+
```

Comment:
Please remove `not` and swap `then` and `else` parts.

Reply:
likewise, "not PTY" is primary path